package.json

```diff
@@ -35,6 +35,8 @@
     "expo": "^15.1.3",
     "ramda": "^0.23.0",
     "react": "~15.4.2",
-    "react-native": "^0.42.0"
+    "react-native": "^0.42.0",
+    "react-native-router-flux": "^3.38.0",
```

Comment:
To get it run on OS X, I needed to downgrade react-native-router-flux, i.e. `"react-native-router-flux": "3.38.0",` instead of `"react-native-router-flux": "^3.38.0",`

* https://github.com/aksonov/react-native-router-flux/issues/1803
* http://stackoverflow.com/questions/43540811/duplicate-module-name-react-native-vector-icons